oidc-service-impl/src/main/java/io/mosip/esignet/services/OAuthServiceImpl.java

```diff
@@ -110,8 +110,12 @@ public TokenResponse getTokens(TokenRequest tokenRequest,boolean isV2) throws Es
             transaction.setEncryptedKyc(kycExchangeResult.getEncryptedKyc());
             auditWrapper.logAudit(Action.DO_KYC_EXCHANGE, ActionStatus.SUCCESS, AuditHelper.buildAuditDto(transaction.getTransactionId(), transaction), null);
         }
-
-        TokenResponse tokenResponse = getTokenResponse(transaction, isTransactionVCScoped);
+        Optional<Integer> consentExpireSeconds = Optional.empty();
+        JsonNode additionalConfig = clientDetailDto.getAdditionalConfig();
+        if(additionalConfig != null && additionalConfig.get("consent_expire_in_mins") != null) {
+            consentExpireSeconds = Optional.of(additionalConfig.get("consent_expire_in_mins").intValue()*60);
```

Comment:
Create a constant for "consent_expire_in_mins" instead of defining the string everywhere.